wiki/History_of_osu!/Online_rankings/osu!/en.md

```diff
@@ -18,7 +18,7 @@ Over the years, there were three different systems used for ranking players:
 
 - [Ranked Score](/wiki/Gameplay/Score/Ranked_score) (6 October 2007 – 24 July 2012)
 - [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
-- [ppv2](/wiki/Performance_points) (27 January 2014 – present)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – 2024-07-21)
```

Comment:
I'm not sure if this is related to online rankings at all...

Reply:
Should stay at `present` though

Reply:
pp system didn't change, no need to edit